res/css/_common.scss

```
@@ -303,6 +303,72 @@ legend {
     overflow-y: auto;
 }
 
+.mx_Dialog .markdown-body {
```

I feel like we're copy/pasting these styles from somewhere - can we just add the `mx_Dialog` targets to that place? (or pull the markdown changes to this file?)

---

They're copied from [_EventTile.scss](https://github.com/matrix-org/matrix-react-sdk/blob/develop/res/css/views/rooms/_EventTile.scss#L466), yes. There are also bits of `markdown-body` styling in `_GroupLayout.scss` and `_ReplyTile.scss`. I could try and extract the styling from `_EventTile.scss` into `_common.scss`? I'm not sure if it wouldn't be better to keep them separate for the future though.

---

a comment to say that they're copied should be sufficient, I think. We *should* extract them properly, but I've been enough of a blocker on this PR already. 

```suggestion
// Styles copied/inspired by GroupLayout, ReplyTile, and EventTile variants.
.mx_Dialog .markdown-body {
```

---

Nah, you haven't been a blocker at all. If anything I should have done things right from the start. 🙈